src/main/java/homework15/lib/Injector.java

```diff
@@ -0,0 +1,40 @@
+package homework15.lib;
+
+import homework15.controller.ConsoleHandler;
+import homework15.dao.BetDao;
+import homework15.dao.BetDaoImpl;
+import homework15.dao.OrderDao;
+import homework15.dao.OrderDaoImpl;
+import homework15.factory.BetDaoFactory;
+import homework15.factory.OrderDaoFactory;
+
+import java.lang.reflect.Field;
+
+public class Injector {
+
+    public static void injectDependency() throws IllegalAccessException {
+        Class<ConsoleHandler> consoleHandlerClass = ConsoleHandler.class;
+        Class<BetDaoImpl> betDaoImplClass = BetDaoImpl.class;
+        Class<OrderDaoImpl> orderDaoImplClass = OrderDaoImpl.class;
+
+        if (betDaoImplClass.getDeclaredAnnotation(Dao.class) != null) {
```

Comment:
maybe we need check Inject.class here?

Reply:
no. because we checked annotation in betDaoImpl class.